### PR TITLE
Fix for RemoveUneeded spaces

### PR DESCRIPTION
**Pull request by ivandrofly**

### CODE REVIEW

**Comment by XhmikosR** [comment]
We need tests...

**Comment by ivandrofly** [comment]
@XhmikosR I comment the example in *Files changed*

**Comment by XhmikosR** [comment]
Sure, but that isn't tests.

**Comment by ivandrofly** [comment]
@XhmikosR https://github.com/ivandrofly/subtitleedit/commit/de47fb1058ed58fd1ecd18af5cf467b56594cc57

File: src/Test/FixCommonErrorsTest.cs [hunk 1]
```diff
@@ -748,6 +748,18 @@ public void FixUnneededSpaces4()
             }
         }
 
+        [TestMethod]
+        public void FixUneededSpaces5()
+        {
+            using(var target = GetFixCommonErrorsLib())
```
Comment (XhmikosR) [inline src/Test/FixCommonErrorsTest.cs]: Missing space after `using`...

**Comment by ivandrofly** [comment]
@XhmikosR https://github.com/ivandrofly/subtitleedit/commit/42ad2650058277175ec7510647e786a86593e035

**Comment by XhmikosR** [comment]
Cool. How about you squash the patches into one?

**Comment by XhmikosR** [comment]
`git rebase -i upstream/master`....

**Comment by ivandrofly** [comment]
#388

**Pull request closed without merging**